### ISSUE TITLE
UnknownOperationException: Unknown operation: MaxPool

### ISSUE THREAD

**Issue by yangta1995** [riga/tfdeploy#12]
The error exists but I do not know why it happened.
I used tf.nn.max_pool(img, ksize=[1,2,2,1], strides = [1,2,2,1],padding='SAME') for MaxPool Operation
Could you give me some advice?

**Comment by riga** [comment]
Hi @yangta1995,

pooling ops are not yet implemented. Maybe I can get some time next week to work on it. But feel free to submit a PR =)

**Comment by riga** [comment]
Duplicate of #3.
